packages/keyring-controller/src/KeyringController.ts

```diff
@@ -658,18 +679,25 @@ export class KeyringController extends BaseController<
    * Adds a new account to the default (first) HD seed phrase keyring.
    *
    * @param accountCount - Number of accounts before adding a new one, used to
+   * @param keyringId - The id of the keyring to add the account to.
    * make the method idempotent.
```

Comment:
```suggestion
   * Adds a new account to the specified keyring
   * or to the default (first) HD seed phrase keyring if none is specified.
   *
   * @param accountCount - Number of accounts before adding a new one, used to
   * make the method idempotent.
   * @param keyringId - The id of the keyring to add the account to.
```

Reply:
We decided not to change that function, so we can keep the description as it was. Good call though! 🙏